### PR TITLE
fix: proofreading pass on 2024 about

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## Description
Fixes up a few typos and a duplicate sentence.

## Related Issue

I didn't file an issue, please forgive me 🙏 

## Motivation and Context
I have no chill.

## How Has This Been Tested?
👀

### CODE REVIEW

File: apps/aot/src/app/about/page.tsx [hunk 1]
```diff
@@ -25,9 +25,7 @@ export default function AboutPage() {
             team is thrilled to unveil our brand-new platform, designed not just for this year's
             event but for many more to come. We hope you enjoy the challenges, learn something new
             along the way, and have a great time participating. If you’re enjoying the experience,
-            we’d love it if you shared the event with your friends to help it grow even more! If you
-            have any questions, spot any bugs, or want to share feedback, don’t hesitate to connect
-            with us on Discord or GitHub. We're here to help!
+            we’d love it if you shared the event with your friends to help it grow even more!
```
Comment (JoshuaKGoldberg) [inline apps/aot/src/app/about/page.tsx:28]: This sentence is a near-duplicate of the one following it.

File: apps/aot/src/app/about/page.tsx [hunk 2]
```diff
@@ -94,17 +92,17 @@ export default function AboutPage() {
             >
               TypeScript Playground
             </a>{' '}
-            preferred over our editor(at least for now 🤐). Just copy the code from the test
-            section, start coding, and when you nail it, pop your code (except the tests) back into
-            our editor.
+            preferred over our editor (at least for now 🤐). You can copy the code from the test
```
Comment (JoshuaKGoldberg) [inline apps/aot/src/app/about/page.tsx:95]: > Just

https://tipsforteachers.substack.com/p/59-stop-saying-the-word-just?open=false#%C2%A7a-tip-to-try-in-class-this-week

File: apps/aot/src/app/about/page.tsx [hunk 3]
```diff
@@ -94,17 +92,17 @@ export default function AboutPage() {
             >
               TypeScript Playground
             </a>{' '}
-            preferred over our editor(at least for now 🤐). Just copy the code from the test
```
Comment (JoshuaKGoldberg) [inline apps/aot/src/app/about/page.tsx:97]: > Just

https://tipsforteachers.substack.com/p/59-stop-saying-the-word-just?open=false#%C2%A7a-tip-to-try-in-class-this-week

**Review by bautistaaa — APPROVED** [review]
oh thats def better 🤣

File: apps/aot/src/app/about/page.tsx (hunk 2 above)
Comment (JoshuaKGoldberg) [inline apps/aot/src/app/about/page.tsx:95]: > into the playground

Clarity: it wasn't 100% explicit beforehand where to copy code to, and I got a little confused myself.

File: apps/aot/src/app/about/page.tsx [hunk 4]
```diff
@@ -94,17 +92,17 @@ export default function AboutPage() {
...
+            any tests) back into our editor.
           </li>
           <li>
             The test cases are your friends! They’ll guide you through what we’re asking for. When
             things get trickier, these cases will be your trusty helpers.
           </li>
           <li>
-            Join the advent-of-typescript channel in our discord linked above just for chewing over
-            these challenges. Jump in, chat, and share your thoughts on solutions. We highly
+            Join the advent-of-typescript channel in our discord linked earlier just for chewing
```
Comment (JoshuaKGoldberg) [inline apps/aot/src/app/about/page.tsx:104]: > above

Accessibility side note: it's not "above" if you're not visually lookinga t the page.

**Pull request merged**